Frontend/src/api/types.d.ts

```diff
@@ -0,0 +1,41 @@
+// These will be generated by the swagger file generated with Rswag
+type WCAEvent =
```

Comment:
Can we try to use `@WCA/Helpers` for this project? It seems like a no brainer.

Reply:
To clarify, `@WCA/Helpers` already has an `EventId` type it exports.

Reply:
ah cool, will use